Makefile

```diff
@@ -93,4 +93,7 @@ indexer-v-algod-swagger:
 
 indexer-v-algod: nightly-setup indexer-v-algod-swagger nightly-teardown
 
+update-submodule:
```

Comment:
Is a reversal of this command needed as well to bring back the git-submodule to its previous state so that other follow-up tests on the same box work as expected? If this is for the very last job, it's not as much of an issue. But to be extra careful you could add a companion that reverts the update. E.g. `make nightly-setup` with its companion `make nightly-teardown`.

Reply:
also consider a slightly more informative name such as `rel-nightly-update-submodule` or similar

Reply:
this target is not for nightly run. it's meant to be ran manually when the submodule needs to be updated. I think naming it `rel-nightly-update-submodule` would be confusing.